### PR TITLE
Display headings

### PR DESCRIPTION
**Pull request by mojulia**

### CODE REVIEW

File: modules/campaignion_petition/campaignion_petition.features.field_instance.inc [hunk 1]
```diff
@@ -507,7 +507,62 @@ function campaignion_petition_field_default_field_instances() {
...
+        'label' => 'above',
+        'settings' => array(),
+        'type' => 'hidden',
+        'weight' => 0,
+      ),
+      'default' => array(
+        'label' => 'above',
+        'module' => 'text',
+        'settings' => array(),
+        'type' => 'text_default',
```
Comment (katzmo) [inline modules/campaignion_petition/campaignion_petition.features.field_instance.inc]: The field should not be displayed in default view mode.

File: modules/campaignion_flexible_form/campaignion_flexible_form.features.field_instance.inc [hunk 2]
```diff
@@ -104,7 +104,7 @@ function campaignion_flexible_form_field_default_field_instances() {
         'summary_rows' => 5,
       ),
       'type' => 'text_textarea_with_summary',
-      'weight' => 5,
+      'weight' => 15,
```
Comment (katzmo) [inline modules/campaignion_flexible_form/campaignion_flexible_form.features.field_instance.inc]: I don’t see any reason for the weights in the content group to change? Would prefer to keep the diff as minimal as possible. If I haven’t overlooked something, only group weights and the pgbar field weight need to be updated on all content types.

**Review by katzmo — CHANGES_REQUESTED** [review]

File: modules/campaignion_petition/campaignion_petition.features.field_instance.inc (hunk 1 above)
Comment (mojulia) [reply modules/campaignion_petition/campaignion_petition.features.field_instance.inc]: ah updated and re-exported it :)

File: modules/campaignion_flexible_form/campaignion_flexible_form.features.field_instance.inc (hunk 2 above)
Comment (mojulia) [reply modules/campaignion_flexible_form/campaignion_flexible_form.features.field_instance.inc]: I _think_ I double checked now which weights should change and which shouldn't but happy if you take a close look again :eyes:

Comment (katzmo) [reply modules/campaignion_flexible_form/campaignion_flexible_form.features.field_instance.inc]: Thanks!

For campaigns, I think the new field should be in `group_only_published` as it only makes sense when the campaign has an actual page → then the description field doesn’t change.

The change in `node-email_to_target-field_no_target_message` and `node-email_to_target-field_email_to_target_options` looks justified.

Comment (mojulia) [reply modules/campaignion_flexible_form/campaignion_flexible_form.features.field_instance.inc]: cooli, have exported it again :)

**Review by katzmo — APPROVED** [review]

**Pull request merged**